src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/ErrorContent.cs

```diff
@@ -18,31 +17,34 @@ public class ErrorContent : AIContent
     /// <summary>The error message.</summary>
     private string _message;
 
-    /// <summary>Initializes a new instance of the <see cref="ErrorContent"/> class with the specified message.</summary>
-    /// <param name="message">The message to store in this content.</param>
+    /// <summary>Initializes a new instance of the <see cref="ErrorContent"/> class with the specified error message.</summary>
+    /// <param name="message">The error message to store in this content.</param>
     [JsonConstructor]
     public ErrorContent(string message)
     {
-        _message = Throw.IfNull(message);
+        // The message shouldn't be null. However, if it's being constructed from another system that may end up
+        // producing a null message, we want to be lenient and not produce an exception while trying to store
+        // information about an error. So, rather than throwing if message is null, we just normalize to empty.
+        _message = message ?? string.Empty;
     }
 
     /// <summary>Gets or sets the error message.</summary>
     public string Message
     {
         get => _message;
-        set => _message = Throw.IfNull(value);
+        set => _message = value ?? string.Empty;
```

Comment:
Should we add [AllowNull] to emphasize the change?

Reply:
If we want to do that, we'd also want to add `?` to the ctor. I don't have a strong preference either way. TextContent explicitly allows nulls, so I guess it'd make sense to be consistent and do the same here.